cmd/image-builder/repos.go

```diff
@@ -19,7 +21,41 @@ var defaultDataDirs = []string{
 	"/usr/share/image-builder",
 }
 
-var newRepoRegistry = func(dataDir string) (*reporegistry.RepoRegistry, error) {
+type repoConfig struct {
+	DataDir    string
+	ExtraRepos []string
+}
+
+func parseExtraRepo(extraRepo string) ([]rpmmd.RepoConfig, error) {
+	// XXX: we want to eventually support more repos here:
+	// - copr:@osbuild/osbuild (with full gpg retrival via the API)
+	// - https://example.com/repo/fedora-42-x86_64/ (with sslverify=True)
+	// But for now we only support local dirs
```

Comment:
Should we then require it be prefixed with `file://` (or `path://`)?

Reply:
I agree that requiring the fule URL would make much more sense and make things easier on our side. As a nice benefit, `https://` repos would start working out of the box...

Reply:
Sure thing, this is updated now.